pkg/lint/list.go

```diff
@@ -153,6 +153,22 @@ func GetRules(fs afero.Fs, finder repoFinder, excludeWarnings bool, parser sqlPa
 			AssetValidator:   ValidateDuplicateColumnNames,
 			ApplicableLevels: []Level{LevelPipeline, LevelAsset},
 		},
+		&SimpleRule{
+			Identifier:       "duplicate-column-checks",
```

Comment:
@karakanb @albertobruin I think `duplicate-column-checks` is not required, wdyt ?